src/Components/WebAssembly/WebAssembly.Authentication/src/Models/RemoteAuthenticationState.cs

```diff
@@ -13,5 +15,10 @@ public class RemoteAuthenticationState
         /// It must be a url within the page.
         /// </summary>
         public string ReturnUrl { get; set; }
+
+        /// <summary>
+        /// Gets or sets additional query parameters.
+        /// </summary>
+        public Dictionary<string, string> ExtraQueryParameters { get; set; }
```

Comment:
The right place for this is the MSAL provider options, this is not a feature all our implementations offer and not something we plan to do at the moment.